docs/source/datasets/index.rst

```diff
@@ -42,80 +42,123 @@ The following simulant attributes are included in this dataset:
    :header-rows: 1
 
    * - Attribute Name
-     - Column Name    
+     - Column Name
+     - Notes    
    * - Unique simulant ID
      - :code:`simulant_id`
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - First name
      - :code:`first_name`
+     - 
    * - Middle initial
      - :code:`middle_initial`
+     - 
    * - Last name
      - :code:`last_name`
+     - 
    * - Age
-     - :code:`age`  
+     - :code:`age` 
+     - Rounded to nearest integer. 
    * - Date of birth
      - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
    * - Physical address street number
      - :code:`street_number`
+     - 
    * - Physical address street name
      - :code:`street_name`
+     - 
    * - Physical address unit
      - :code:`unit_number`
+     - 
    * - Physical address city
-     - :code:`city`    
+     - :code:`city` 
+     -    
    * - Physical address state
      - :code:`state`  
+     - 
    * - Physical address ZIP code
      - :code:`zipcode`
+     - 
    * - Relationship to person 1 (head of household)
      - :code:`relationship_to_household_head` 
-   * - Sex (binary; 'male' or 'female')
+     - 'Person 1', 'head of household', and 'Reference person' are all synonymous in this context. Possible values for this indicator include:
```

Comment:
shoot, should we change this to "reference_person" to be consistent?

Reply:
I'd be in favor of this, but not sure how difficult it would be to change.